dir.props

```diff
@@ -2,6 +2,12 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Condition="Exists('..\dir.props')" Project="..\dir.props" />
 
+  <!-- We shipped assembly file version 4.6.x up until the end of rc3.  Version assembly as 
+        4.6.x to ensure compatability in Visual Studio for an in-place update. -->
+  <PropertyGroup>
+    <MajorVersion>4</MajorVersion>
+    <MinorVersion>6</MinorVersion>
```

Comment:
Seems that it is based on a single spot check of RC2 vs latest.  I still think it's unusual to use 4.6 here but I don't I guess it's just as good as any arbitrary number >= 4.6.